src/utils.rs

```diff
@@ -65,13 +54,15 @@ pub fn print_vulnerability<S: AsRef<str>>(text: S, criticality: Criticality) {
         let message = format!("Possible {} criticality vulnerability found!: {}",
                               criticality,
                               text.as_ref());
-        println!("{}",
-                 match criticality {
-                     Criticality::Low => message.cyan(),
-                     Criticality::Medium => message.yellow(),
-                     Criticality::High | Criticality::Critical => message.red(),
-                     _ => return,
-                 });
+
+         let formatted_message = match criticality {
+             Criticality::Low => message.cyan(),
+             Criticality::Medium => message.yellow(),
+             Criticality::High | Criticality::Critical => message.red(),
+             _ => return,
+         };
+
+        println!("{}", formatted_message);
```

Comment:
This should only be printed in `debug` mode, even though I think we are doing the filtering before everything else. We could in any case test it inside the function and avoid a lot of boilerplate code.